docker-compose.yml

```diff
@@ -70,6 +70,11 @@ services:
       - type: bind
         source: ./app
         target: /app/
+      - type: bind
```

Comment:
We shouldn't bind the docker socket here, could you run through celery_evaluation_worker instead?